chat/src/main/java/net/md_5/bungee/api/chat/SelectorComponent.java

```diff
@@ -0,0 +1,50 @@
+package net.md_5.bungee.api.chat;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.ToString;
+
+/**
+ * This component processes a target selector into a pre-formatted set of discovered names.<br>
+ * Multiple targets may be obtained, and with commas seperating each one and a final "and" for the last target.
+ * The resulting format cannot be overwritten. This includes all styling from team prefixes, insertions, click events, and hover events.<br>
+ * A bug (MC-53673) currently prevents full usage within hover events.
+ */
```

Comment:
Maybe make a note that these (and scores?) are filled in server side by the implementation.

BungeeCord does have a scoreboard API, so it would be desirable to at least sketch an implementation of doing that required processing for scoreboard components.

Reply:
That's easy enough to do (evaluate score components) if the player's name is provided, the * wildcard is used, or if it's a dummy player. However, if a selector is passed in then it basically becomes impossible to do within BungeeCord.

Reply:
BungeeCord doesn't know about any values for scores.